### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by kazemnejad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,8 +79,70 @@ git push --set-upstream origin my_awesome_task
 git commit -m "Submission for my_awesome_task"
 git push
 ```
+### Task
+GenBench task involves providing a test set along with optional train and validation sets. Each set is comprised of a collection of examples, where each example is represented as a dictionary containing the input and the corresponding ground truth output.
```
Comment (christos-c) [inline README.md]: "A GenBench task"

File: README.md [hunk 2]
```diff
@@ -79,8 +79,70 @@ git push --set-upstream origin my_awesome_task
...
+
+It's worth noting that we offer complete flexibility in terms of how these examples are prepared and how the model's predictions are evaluated. This allows for the creation of diverse and varied benchmarking that can be tailored to the needs of specific generalization protocols.
```
Comment (christos-c) [inline README.md]: ~It's worth noting that~ We offer complete flexibility in terms of how these examples are prepared and how the model's predictions are evaluated.

File: README.md [hunk 3]
```diff
@@ -162,6 +224,7 @@ To this end, GenBench tasks exist as Python classes and are defined using two fi
     // Multiple-choice tasks:
```
Comment (christos-c) [inline README.md:224]: The fields should be Input, Target and Target Options, right? (I made the error originally).

File: genbench/tasks/sample_task/config.jsonnet [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    task_type: 'free_form',
+
+    evaluation_metrics: [
+        {
+            type: 'hf',
+            path: ['EM'],
+        }
+    ],
+
+    model_prepation_strategies: {
```
Comment (christos-c) [inline genbench/tasks/sample_task/config.jsonnet]: Should be `preparation_strategies`

**Review by christos-c — APPROVED** [review]
